File: Programmers/Python/Level1/Arranging_strings_in_descending_order.py
def solution(s):
    answer = []
    for i in range(len(s)):
        answer.append(s[i])
        
    answer.sort(reverse=True)
    a = ''.join(answer)
    return a

    
#def solution(s):
#    return ''.join(sorted(s, reverse=True))

# ''.join()을 사용하면 list의 요소들을 띄워쓰기 없이 string으로 변환해준다.
# sorted()함수에서 reverse=True를 해주면 내림차순으로 정렬된다.
# sorted()함수의 매개변수에 list가 아닌 string을 넣어도 된다. ( 단, 반환은 list로 된다. )

File: Programmers/C++/Level1/Arranging_strings_in_descending_order.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s) {
    sort(s.begin(), s.end(), greater<int>());   
    return s;
}

// sort()에서 3번째 파라미터에 "greater<자료형>()"을 넣어주면 내림차순으로 정렬한다.